deltachat-ios/Controller/FullMessageViewController.swift

```diff
@@ -76,14 +83,15 @@ class FullMessageViewController: WebViewViewController {
         let alert = UIAlertController(title: String.localized("load_remote_content"),
                                       message: String.localized("load_remote_content_ask"),
                                       preferredStyle: .safeActionSheet)
-        let alwaysAction = UIAlertAction(title: "\(alwaysCheckmark)\(String.localized("always"))", style: .default, handler: alwaysActionPressed(_:))
-        let neverAction = UIAlertAction(title: "\(neverCheckmark)\(String.localized("never"))", style: .default, handler: neverActionPressed(_:))
-        let onceAction = UIAlertAction(title: "\(onceCheckmark)\(String.localized("once"))", style: .default, handler: onceActionPressed(_:))
-
+        if isContactRequest {
+            alert.addAction(UIAlertAction(title: "\(neverCheckmark)\(String.localized("no"))", style: .default, handler: neverActionPressed(_:)))
```

Comment:
If I read this block correctly, it can be deduplicated a bit further while still keeping readable code. 
e.g.
```
let reject = "\(neverCheckmark)\(String.localized(isContactRequest ? "no" : "never"))"
... adding defaults
if isContactRequest {
... adding last action
}

Reply:
i avoided that initially as i do not wanted to have two `isContactRequest` conditions, however, i agree, this is a bit smarter overall, i added a commit. thanks!